packages/cozy-client/src/devtools/Pouch.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useMemo, useState, useCallback, useEffect } from 'react'
+import CozyPouchLink from 'cozy-pouch-link'
+import get from 'lodash/get'
+import Button from 'cozy-ui/transpiled/react/MuiCozyTheme/Buttons'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import format from 'date-fns/format'
+
+import useClient from '../hooks/useClient'
+import PanelContent from './PanelContent'
+
+const listStyle = { paddingLeft: '1rem' }
+
+const DATE_FORMAT = 'HH:mm:ss DD/MM/YYYY'
+
+const useForceUpdate = () => {
+  const [, setLastRender] = useState(null)
+  const forceUpdate = useCallback(() => {
+    setLastRender(Date.now())
+  }, [setLastRender])
```

Comment:
Out of curiosity, what is the purpose of using `useCallback` here? Won't `setLastRender` remain stable between renders?

Reply:
I've removed the callback ;)

Reply:
:relieved: